News/Helper/ImageHelper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ImageHelper.swift
+//  News
+//
+//  Created by Mohamed Elboraey on 16/05/2025.
+//
+
+import Foundation
+import UIKit
+
+class ImageHelper {
+    static let shared = ImageHelper() // Fixed singleton name
+    
+    func loadImage(from urlString: String,
+                  onLoading: (() -> Void)? = nil,
+                  completion: @escaping (UIImage?) -> Void) {
```

Comment:
```suggestion
                  completion: @escaping (Result<UIImage, Error>) -> Void) {
```
Optional -- A better API design is to return a result, it should be much easier for the caller to understand what's the type of the error being received.